### PR TITLE
ES-906755 -Modify DocIO GitHub examples repository for document processing playground

### PR DESCRIPTION
**Pull request by DharanyaSakthivel-SF4210**
Add-playground-in-sample

### CODE REVIEW

File: Word-document/Merge–multiple-Word-files-in-same-page/.NET/Merge–multiple-Word-files-in-same-page/Program.cs [hunk 1]
```diff
@@ -4,19 +4,19 @@
...
 List<string> sourceFileNames = new List<string>();
-sourceFileNames.Add("../../../Data/Addressblock.docx");
-sourceFileNames.Add("../../../Data/Salutation.docx");
-sourceFileNames.Add("../../../Data/Greetings.docx");
+sourceFileNames.Add("Data/Addressblock.docx");
+sourceFileNames.Add("Data/Salutation.docx");
+sourceFileNames.Add("Data/Greetings.docx");
 
-string destinationFileName = "../../../Data/Title.docx";
+string destinationFileName = "Data/Title.docx";
```
Comment (LokeshBaskar) [inline Word-document/Merge–multiple-Word-files-in-same-page/.NET/Merge–multiple-Word-files-in-same-page/Program.cs:11]: Please use the Path.GetFullPath

File: Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿using Syncfusion.DocIO;
 using Syncfusion.DocIO.DLS;
 
-using (FileStream fileStreamPath = new FileStream(@"../../../Data/Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+using (FileStream fileStreamPath = new FileStream(@"Data/Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
Comment (LokeshBaskar) [inline Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs:4]: Please use the Path.GetFullPath

File: Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 
 //Opens an existing Word document
-using (FileStream inputStream = new FileStream("../../../Data/Sample.docx", FileMode.Open, FileAccess.Read))
+using (FileStream inputStream = new FileStream("Data/Sample.docx", FileMode.Open, FileAccess.Read))
```
Comment (LokeshBaskar) [inline Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs:6]: Please use the Path.GetFullPath

File: Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs [hunk 4]
```diff
@@ -57,7 +57,7 @@ static void Main(string[] args)
                     if (newDocument != null)
                     {
                         //Saves the Word document
-                        string fileName = @"../../../Document" + (headingIndex + 1) + ".docx";
+                        string fileName = @"Output/Document" + (headingIndex + 1) + ".docx";
```
Comment (LokeshBaskar) [inline Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs:60]: Please use the Path.GetFullPath

File: Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs [hunk 5]
```diff
@@ -37,7 +37,7 @@ static void Main(string[] args)
                                     if (newDocument != null)
                                     {
                                         //Saves the Word document
-                                        string fileName = @"../../../Document" + (headingIndex + 1) + ".docx";
+                                        string fileName = @"Output/Document" + (headingIndex + 1) + ".docx";
```
Comment (LokeshBaskar) [inline Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs:40]: Please use the Path.GetFullPath

File: Word-document/Split-by-section/.NET/Split-by-section/Program.cs [hunk 6]
```diff
@@ -23,7 +23,7 @@ static void Main(string[] args)
                             //Add cloned section into new Word document
                             newDocument.Sections.Add(section.Clone());
                             //Saves the Word document to  MemoryStream
-                            using (FileStream outputStream = new FileStream(@"../../../Section" + fileId + ".docx", FileMode.OpenOrCreate, FileAccess.ReadWrite))
+                            using (FileStream outputStream = new FileStream(@"Output/Section" + fileId + ".docx", FileMode.OpenOrCreate, FileAccess.ReadWrite))
```
Comment (LokeshBaskar) [inline Word-document/Split-by-section/.NET/Split-by-section/Program.cs:26]: Please use the Path.GetFullPath

File: Table-Of-Contents/Add_toc_in_merged_documents/.NET/Add_toc_in_merged_documents/Program.cs [hunk 7]
```diff
@@ -3,7 +3,7 @@
 using Syncfusion.DocIORenderer;
 
 //Get the Source document names from the folder.
-string[] sourceDocumentNames = Directory.GetFiles(@"../../../Data/");
+string[] sourceDocumentNames = Directory.GetFiles(@"Data/");
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Table-Of-Contents/Add_toc_in_merged_documents/.NET/Add_toc_in_merged_documents/Program.cs:6]: resolved

File: Table-Of-Contents/Add_toc_in_merged_documents/.NET/Add_toc_in_merged_documents/Program.cs [hunk 8]
```diff
@@ -34,7 +34,7 @@
     //Updates the table of contents
     destinationDocument.UpdateTableOfContents();
     //Save the destination document.
-    using (FileStream outputStream = new FileStream(@"../../../Output.docx", FileMode.Create, FileAccess.Write))
+    using (FileStream outputStream = new FileStream(@"Output/Output.docx", FileMode.Create, FileAccess.Write))
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Table-Of-Contents/Add_toc_in_merged_documents/.NET/Add_toc_in_merged_documents/Program.cs:37]: resolved

File: Word-document/Apply-picture-background-to-document/.NET/Apply-picture-background-to-document/Program.cs [hunk 9]
```diff
@@ -8,15 +8,15 @@ class Program
...
+            using (FileStream fileStreamPath = new FileStream(Path.GetFullPath(@"Data/Template.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Opens an document from file system through constructor of WordDocument class.
                 using (WordDocument document = new WordDocument(fileStreamPath, FormatType.Automatic))
                 {
                     //Sets the background type as picture.
                     document.Background.Type = BackgroundType.Picture;
                     //Opens the existing image. 
-                    using (FileStream imageStream = new FileStream(@"../../../Data/Picture.png", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (FileStream imageStream = new FileStream(@"Data/Picture.png", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Apply-picture-background-to-document/.NET/Apply-picture-background-to-document/Program.cs:19]: modified

File: Word-document/Copy-Excel-content-to-Word-with-formatting/.NET/Copy-Excel-content-to-Word-with-formatting/Program.cs [hunk 10]
```diff
@@ -35,7 +35,7 @@ void ExtractExcelContent(WTable table)
     IApplication application = excelEngine.Excel;
     application.DefaultVersion = ExcelVersion.Xlsx;
     //Load the file into stream
-    FileStream inputExcelStream = new FileStream("../../../Data/Sample.xlsx", FileMode.Open, FileAccess.Read);
+    FileStream inputExcelStream = new FileStream("Data/Sample.xlsx", FileMode.Open, FileAccess.Read);
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Copy-Excel-content-to-Word-with-formatting/.NET/Copy-Excel-content-to-Word-with-formatting/Program.cs:38]: resolved

File: Word-document/Copy-Excel-content-to-Word-with-formatting/.NET/Copy-Excel-content-to-Word-with-formatting/Program.cs [hunk 11]
```diff
@@ -18,7 +18,7 @@
 ExtractExcelContent(table);
 
 //Load the file into stream
-FileStream outputStream = new FileStream("../../../Data/Output.docx", FileMode.Create, FileAccess.Write);
+FileStream outputStream = new FileStream("Output/Output.docx", FileMode.Create, FileAccess.Write);
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Copy-Excel-content-to-Word-with-formatting/.NET/Copy-Excel-content-to-Word-with-formatting/Program.cs:21]: resolved

File: Word-document/Find-and-replace-text-with-Excel-content/.NET/Find-and-replace-text-with-Excel-content/Program.cs [hunk 12]
```diff
@@ -24,7 +24,7 @@
         document.Replace("<<ExcelPlaceHolder>>", bodyPart, true, true, false);
 
         //Load the file into stream
-        FileStream outputStream = new FileStream("../../../Data/Output.docx", FileMode.Create, FileAccess.Write);
+        FileStream outputStream = new FileStream("Output/Output.docx", FileMode.Create, FileAccess.Write);
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Find-and-replace-text-with-Excel-content/.NET/Find-and-replace-text-with-Excel-content/Program.cs:27]: resolved

File: Word-document/Find-and-replace-text-with-Excel-content/.NET/Find-and-replace-text-with-Excel-content/Program.cs [hunk 13]
```diff
@@ -40,7 +40,7 @@ void ExtractExcelContent(WTable table)
     IApplication application = excelEngine.Excel;
     application.DefaultVersion = ExcelVersion.Xlsx;
     //Load the file into stream
-    FileStream inputExcelStream = new FileStream("../../../Data/Sample.xlsx", FileMode.Open, FileAccess.Read);
+    FileStream inputExcelStream = new FileStream("Data/Sample.xlsx", FileMode.Open, FileAccess.Read);
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Find-and-replace-text-with-Excel-content/.NET/Find-and-replace-text-with-Excel-content/Program.cs:43]: resolved

File: Word-document/Merge-documents-with-same-header-and-footer/.NET/Merge-documents-with-same-header-and-footer/Program.cs [hunk 14]
```diff
@@ -12,13 +12,13 @@ class Program
...
             //Open the destination document as a stream.
-            using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Open the destination document.
                 using (WordDocument destinationDocument = new WordDocument(destinationStreamPath, FormatType.Automatic))
                 {
                     //Get the Source document names from the folder.
-                    string[] sourceDocumentNames = Directory.GetFiles(@"../../../Data/SourceDocuments/");
+                    string[] sourceDocumentNames = Directory.GetFiles(@"Data/SourceDocuments/");
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Merge-documents-with-same-header-and-footer/.NET/Merge-documents-with-same-header-and-footer/Program.cs:21]: resolved

File: Word-document/Merge-without-changing-page-numbers/.NET/Merge-without-changing-page-numbers/Program.cs [hunk 15]
```diff
@@ -11,13 +11,13 @@ class Program
...
             //Open the file as a stream.
-            using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Load the file stream into a Word document.
                 using (WordDocument destinationDocument = new WordDocument(destinationStreamPath, FormatType.Automatic))
                 {
                     //Get the Source document names from the folder.
-                    string[] sourceDocumentNames = Directory.GetFiles(@"../../../Data/SourceDocuments/");
+                    string[] sourceDocumentNames = Directory.GetFiles(@"Data/SourceDocuments/");
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Merge-without-changing-page-numbers/.NET/Merge-without-changing-page-numbers/Program.cs:20]: resolved

File: Word-document/Merge–multiple-Word-files-in-same-page/.NET/Merge–multiple-Word-files-in-same-page/Program.cs (hunk 1 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Merge–multiple-Word-files-in-same-page/.NET/Merge–multiple-Word-files-in-same-page/Program.cs:11]: resolved

File: Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs (hunk 2 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs:4]: resolved

File: Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs [hunk 16]
```diff
@@ -12,7 +12,7 @@
         ChangeParagraphFormatting(document);
         //Change table style formatting
         ChangeTableFormatting(document);
-        using (FileStream outputStream = new FileStream(@"../../../Data/Output.docx", FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite))
+        using (FileStream outputStream = new FileStream(@"Output/Output.docx", FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite))
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Modify-formatting-for-default-styles/.NET/Modify-formatting-for-default-styles/Program.cs:15]: resolved

File: Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs (hunk 3 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs:6]: resolved

File: Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs [hunk 17]
```diff
@@ -22,7 +22,7 @@
                 document.ChildEntities.RemoveAt(SectionIndex);
             }
         }
-        using (FileStream outputStream = new FileStream("../../../Data/Output.docx", FileMode.Create, FileAccess.Write))
+        using (FileStream outputStream = new FileStream("Output/Output.docx", FileMode.Create, FileAccess.Write))
```
[the remark this answers is not in the document]
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Remove-empty-pages-from-Word-document/.NET/Remove-empty-pages-from-Word-document/Program.cs:25]: Resolved

File: Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs (hunk 4 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs]: modified

File: Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs (hunk 5 above)
Comment (DharanyaSakthivel-SF4210) [reply Word-document/Split-by-heading/.NET/Split-by-heading/Program.cs]: modified

**Review by LokeshBaskar — APPROVED** [review]
No concerns

**Pull request merged**